espnet2/tasks/abs_task.py

```diff
@@ -840,14 +888,75 @@ def main(cls, args: argparse.Namespace = None, cmd: Sequence[str] = None) -> Non
                 nbest=args.keep_n_best_checkpoints,
             )
 
+    @classmethod
+    def build_iter_factory(
+        cls,
+        args: argparse.Namespace,
```

Comment:
As I mentioned above, args is a little bit confusing.
Is it ok to convert args to separated keyword arguments?